main.js

```diff
@@ -1,12 +1,49 @@
 'use strict';
 
-const logger = require('ft-next-logger');
+const logger = require('ft-next-logger').logger;
+const fetchres = require('fetchres');
+const raven = require('raven');
+let ravenMiddleware;
+
+function sendErrorDev (err, req, res) {
+	if (err.name === fetchres.ReadTimeoutError.name) {
+		logger.error('Dependency timeout -', err);
```

Comment:
I had a feeling that was the case. Does that work with an arbitrary list of objects e.g. `error('oops', {custom: 'prop'}, err)`